SPECS/packer/packer.spec

```diff
@@ -31,14 +31,11 @@ Source0:        https://github.com/hashicorp/packer/archive/refs/tags/v%{version
 #       - The additional options enable generation of a tarball with the same hash every time regardless of the environment.
 #         See: https://reproducible-builds.org/docs/archives/
 #       - For the value of "--mtime" use the date "2021-04-26 00:00Z" to simplify future updates.
-Source1:        %{name}-%{version}-vendor.tar.gz
-Patch0:         CVE-2023-45288.patch
-Patch1:         CVE-2022-3064.patch
-Patch2:         CVE-2023-49569.patch
-Patch3:         CVE-2024-6104.patch
-Patch4:         CVE-2024-24786.patch
-Patch5:         CVE-2024-45337.patch
-Patch6:         CVE-2024-45338.patch
```

Comment:
any reason for the removal of these patches?

Reply:
All these are fixed by version upgrade for the vendors

Reply:
Ok makes sense.